src/wp-includes/blocks.php

```diff
@@ -185,7 +175,16 @@ function register_block_script_handle( $metadata, $field_name, $index = 0 ) {
 		$script_args['strategy'] = 'defer';
 	}
 
-	$script_asset        = require $script_asset_path;
+	if ( file_exists( $script_asset_path ) ) {
+		$script_asset = require $script_asset_path;
+	} else {
+		// Ensure graceful fallback for missing or inaccessible *.asset.php by setting default dependencies and version.
+		$script_asset = array(
```

Comment:
Would it be possible to simplify it to:

```suggestion
		$script_asset = array();
```

given that access to the array is guarded later and the same default values seem to be provided?